server/src/main/java/org/apache/druid/client/BrokerServerView.java

```diff
@@ -322,4 +322,10 @@ private void runTimelineCallbacks(final Function<TimelineCallback, CallbackActio
       );
     }
   }
+
+  @Override
+  public Map<String, QueryableDruidServer> getClients()
+  {
+    return clients;
```

Comment:
The caller needs only `DruidServer`, not entire map or `QueryableDruidServer`. Also, it's much better to not return mutable objects. Please change this to like below:

```java
return clients.values().stream()
           .map(queryableDruidServer -> queryableDruidServer.getServer().toImmutableDruidServer())
           .collect(Collectors.toList());
```

Reply:
good suggestion, changed.